Add Community Builder listing markdown file (#9119)
Couchers-org/couchers: app/web/markdown/volunteer/community-builder.md

Reviewer: Other pages don't have a `-listing` suffix. The `volunteer` folder should be enough for namespacing.

Reviewer: You should reference this document from `app\web\markdown\volunteer.md` so it is findable.

---
title: "Community Builder"
description: "Apply to become a Community Builder for Couchers.org"
---

## [Apply Here](https://forms.monday.com/forms/1e84199578bc028b282a88b9e325d646?r=use1)

## Position Description

**This is a local position**

We are recruiting Community Builders (CBs) around the world to help grow, nurture, and support local Couchers.org communities through active engagement. As a Community Builder, you will serve as an ambassador for Couchers.org. You will not only engage with members in your area, but also communicate with and represent your local community to other CBs.

This is an unpaid volunteer position where you will work closely with the Community Builder Program Managers (CBPMs).

> ### Duties

Reviewer 2: @c-kreuz We don't expect Community Builders to do all of these things. Some CBs are super active and great at organizing events, whereas others maintain a strong online presence and help support the community there and others do it through very actively hosting or surfing as a nomad. 

We should find a way to make it clear what is actually mandatory in terms of duties; the mandatory ones that come to mind are:
* representing Couchers.org in a positive manner that also reflects our values and guidelines
* working with CBPMs (one example: Provide feedback to the CBPM regarding any challenges with the platform or community)
* upholding our rules: https://couchers.org/rules
* actively reporting any violations
* refer people to the proper resources when asked (help center, support team, community page, account settings, etc)

Optional duties, but people should do at least one of them:
* organizing events on a (relatively) regular basis
* very actively hosting / very actively surfing
* Monitor and manage the communities page for their community (update info there, respond to discussions, monitor events/discussions for rule violations, etc)
* welcome new members of the community

- Grow Your Local Community: Drive the creation, growth, and engagement of the Couchers.org platform within your city or region;
- Organize Events: Plan, host, and promote regular events to bring local members and travelers together;
- Welcome & Onboard: Actively welcome new members to your local community and help them get started on the platform;
- Represent & Communicate: Act as the bridge between your local community and the Couchers.org core team, communicating goals, feedback, and local updates;
- Foster Safety & Inclusivity: Monitor community discussions and events, promoting a safe, welcoming, and inclusive environment, and report to the support team if any issues arise.

### Skills and Experience

All applicants must meet the following baseline criteria **before** applying. To ensure our Community Builders are active, verified, and familiar with the platform, they must have:

> - a [fully filled out profile](https://help.couchers.org/hc/couchersorg-help-center/articles/1725919197-how-do-i-create-a-great-profile) (including bio and photos).

Reviewer 2: ```suggestion
- a [fully filled out profile](https://help.couchers.org/hc/couchersorg-help-center/articles/1725919197-how-do-i-create-a-great-profile) (including thoughtful texts on the "about me" and "my home" sections, and a profile photo which meets our [photo guidelines](https://help.couchers.org/hc/couchersorg-help-center/articles/1777852193-test)).
```

> - written comments on the community discussions page.

Reviewer 3: This seems a bit unclear. Do you mean participated in discussions in the couchers.org communities? or events?

> - written comments on the community discussions page.

Reviewer 2: ```suggestion
- interacted with the community on their local Community Page in discussions, events or both.
```

- hosted local events for members and travelers.

> - actively invited new members to join and build up the local community.

Reviewer 3: this is also a bit vague, how would they prove that?

> - actively invited new members to join and build up the local community.

Reviewer 2: ```suggestion
```

> - completed the Strong Verification (SV) process (i.e. have a [SV badge](https://couchers.org/badges/strong_verification)) to help maintain trust and security on the platform. (can be optional in special circumstances)

Reviewer 2: ```suggestion
- if possible, completed the Strong Verification (SV) process (i.e. have a [SV badge](https://couchers.org/badges/strong_verification)) to help maintain trust and security on the platform. Note: We can of course make exceptions to this requirement in special circumstances.
```

An ideal candidate would have the following skills:
- **Couch Surfing Experience:** Passion for hospitality exchange and a solid understanding of the couch surfing community and culture;

> - **Community Mindset:** Experience (or a strong interest) as an [Event Organizer](./event-organizer), Local Guide, Host, or Community Advocate;

Reviewer: I suggest removing the link if we'd be happy with event org experience outside of our platform's event organizer role, which seem so to be the case. We could also skip capitalizing all the roles.

Reviewer 2: I agree with Tristan regarding removing the link, but I do think we should keep the roles capitalized.

> - **Community Mindset:** Experience (or a strong interest) as an [Event Organizer](./event-organizer), Local Guide, Host, or Community Advocate;

Author: ```suggestion
- **Community Mindset:** Experience (or a strong interest) as an Event Organizer, Local Guide, Host, or Community Advocate;
```

- **Event Planning:** Ability to organize and execute engaging get-togethers, meetups, or events;

### Expectations/Commitment

- 2-5 hours per week on average;
- 6 months minimum commitment;

> - Willingness to role-sharing/collaborate with the other CBs and CBPMs;

Reviewer 3: role-sharing sounds strange maybe just say "collaborate with other..."

> - Willingness to role-sharing/collaborate with the other CBs and CBPMs;

Reviewer 2: ```suggestion
- Willingness to actively collaborate with other CBs, CBPMs, and the Safety & Trust Team;
```

- This is a volunteer position (unpaid). Couchers.org is built and run entirely by volunteers.

> ### Apply

Reviewer: Is this logically within "Position Description"?

```suggestion
## Apply
```

> 1. Fill out the [Commnutiy Builder Form](/community-builder-form).

Reviewer: ```suggestion
1. Fill out the [Community Builder Form](/community-builder-form).
```

We are recruiting Community Builders (CBs) around the world to help grow, nurture, and support local Couchers.org communities through active engagement. As a Community Builder, you will serve as an ambassador for Couchers.org. You will not only engage with members in your area, but also communicate with and represent your local community to other CBs.

This is a volunteer position where you will work closely with the Community Builder Program Managers (CBPMs).

### Duties

- Grow Your Local Community: Drive the creation, growth, and engagement of the Couchers.org platform within your city or region;
- Organize Events: Plan, host, and promote regular events (both online and in-person) to bring local members and travelers together;
- Welcome & Onboard: Actively welcome new members to your local community and help them get started on the platform;
- Represent & Communicate: Act as the bridge between your local community and the Couchers.org core team, communicating goals, feedback, and local updates;
- Foster Safety & Inclusivity: Monitor community discussions and events, promoting a safe, welcoming, and inclusive environment, and report to the support team if any issues arise.

> ### Skills and Experience

Reviewer: I was confused as I read this section because the requirements are a very low bar, then it seems that the "ideal candidates" is really what we're looking for, because if someone had only the prereqs but none of the ideal candidate aspects, I don't think we'd take them.

I think specifying a number of comments or events makes it seem more objective than it is, and we wouldn't want people micro-optimizing for that.

I suggest listing our desired experience where each bullet point must minimally be met but we can still judge if overall that compounds to enough experience:

- Filled out profile with Strong Verification
- Familiarity with the Couchers platform (communities, events, discussions)
- Demonstrated experience with hospitality exchange, event organization, or as a local guide
- Familiarity with the local community (fluency in the local language, lived there many years)
- Working proficiency in English

Bonus: Conflict resolution, social media, communication skills

Author: I revised the requirements to be more ambiguous.
We are in need for CBs and might have to give a chance to candidates that don't meet 100% of the ideal criteria, as long as the fulfill certain requirements.
